components/layout/nav/NavItem.es6.js

```diff
@@ -73,4 +78,6 @@ class NavItem extends React.Component<Props> {
   }
 }
 
-export default NavItem
+export default React.forwardRef<Props, HTMLDivElement>((props, ref) => (
```

Comment:
Forwards refs when component is being used so that we can use the ref prop directly.